Dockerfile

```diff
@@ -24,32 +24,39 @@ RUN \
  echo "deb http://ppa.launchpad.net/jcfp/sab-addons/ubuntu bionic main" >> /etc/apt/sources.list.d/sabnzbd.list && \
  echo "deb-src http://ppa.launchpad.net/jcfp/sab-addons/ubuntu bionic main" >> /etc/apt/sources.list.d/sabnzbd.list && \
  echo "**** install packages ****" && \
- if [ -z ${SABNZBD_VERSION+x} ]; then \
-	SABNZBD="sabnzbdplus"; \
- else \
-	SABNZBD_CLEAN=$(echo "${SABNZBD_VERSION}"| sed 's/..$//') && \
-	SABNZBD="sabnzbdplus=${SABNZBD_CLEAN}"; \
- fi && \
  apt-get update && \
  apt-get install -y \
+	jq \
+	libffi-dev \
+	libssl-dev \
 	p7zip-full \
 	par2-tbb \
 	python3 \
 	python3-pip \
-	${SABNZBD} \
+	python3-six \
 	unrar \
 	unzip && \
```

Comment:
`unzip` is no longer needed, we always use `7zip`.

Reply:
thanks, removed unzip